django_server/api_keys/FEATURES.md

```
@@ -0,0 +1,161 @@
+
+## Key Models
+
+### APIKey
+- Developer API keys for programmatic access
+- SHA-256 hashed storage
+- One-time display of raw keys
+- Creator tracking
+
+### BrowserExtensionAPIKey
+- Keys for browser extension authentication
+- User-specific tokens
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)